app/src/main/java/com/android/deskclock/alarms/AlarmActivity.java

```diff
@@ -288,6 +298,26 @@ protected void onPause() {
             mReceiverRegistered = false;
         }
     }
+    
+    private final BroadcastReceiver PowerBtnReceiver = new BroadcastReceiver() {
+        @Override
+        public void onReceive(Context context, Intent intent) {
```

Comment:
The indention in this method is a bit weird. Please re-format. It might also be easier to read if you added things like `if (...) { return }` instead of nesting the `if` statement so many levels.